src/components/payments/paymentView.tsx

```diff
@@ -65,11 +69,25 @@ export default function PaymentsView({
           </div>
         </div>
       </div>
+      {!loadedSuccessfully ? (
+        <div>
+          <Alert className="flex items-center bg-blue-500">
+            <Icons.warning className="mr-2 h-4 w-4" />
+            <AlertTitle>
+              You payment address is not visible on chain yet or we have not
+              indexed it yet, wait a couple of min and refresh this page
```

Comment:
You payment address has not been indexed yet, wait for a moment until we are able to fetch it from onchain data and refresh this page.